libbpf-cargo/src/gen.rs

```diff
@@ -271,7 +271,7 @@ fn gen_skel_map_defs(
             skel,
             r#"
             pub fn {map_name}(&mut self) -> &mut {return_ty} {{
```

Comment:
I wonder if the skeleton should have immutable/mutable getters too. Gut feeling is yeah, for consistency.

Reply:
I'm thinking it should. Have you had a chance to consider the two options I proposed above?

Reply:
Oh whoops, I think I just immediately clicked over to "files changed" without reading